pkg/alertmanager/multitenant.go

```diff
@@ -997,16 +997,13 @@ func (am *MultitenantAlertmanager) ReplicateStateForUser(ctx context.Context, us
 
 		switch resp.Status {
 		case alertmanagerpb.OK:
```

Comment:
nit: I'd remove this case (and `default`) completely, if they don't do anything.